src/components/user-auth/questionnaire/Questionnaire.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react'
+import ProgressBar from './ProgressBar'
+import Status from './Status'
+import Logo from '../../logo/Logo'
+import { Box, Button } from '@mui/material'
+import ArrowBackIcon from '@mui/icons-material/ArrowBack'
+import { commonStyles } from 'styles/commonStyles'
+import Geolocation from './Geolocation'
+
+const RegistrationForm: React.FC = () => {
+  const { classes } = commonStyles()
+
+  const totalSteps = 3
+  const [step, setStep] = useState<number>(1)
+  const [formData, setFormData] = useState<any>({
```

Comment:
Not sure if it temporary page or not -  there should be no <any> since it is against the Typescript purpose. If this file is not an actual pages and is here for demo it's fine then.

Reply:
Yes, it is temporary. But what do you mean by Typescript purpose?

Reply:
Sorry I missed <any> lol <any> should not be used in Typescript files, since Typescript is all about types and using <any> defeats its purpose, typescript helps to catch mistakes early, and using <any> makes it pointless, you are basically canceling typescript in this particular line. If you don't have a predetermined type, you can create a new custom one by adding type or interface